src/main/java/com/ericsson/ei/services/SubscriptionService.java

```diff
@@ -179,19 +206,44 @@ public boolean deleteSubscription(String subscriptionName) throws AccessExceptio
         return subscriptions;
     }
 
+    /**
+     * Returns a boolean indicating that authentication details was provided in the subscription.
+     *
+     * @param authType
+     * @param username
+     * @param password
+     * @return
+     */
+    private boolean isAuthenticationDetailsProvided(String authType, String username,
```

Comment:
This code is duplicated and ashould be moved to a seperate class. Maybe the EncryptionFormatter could be used for this.

Reply:
Moved the method